src/app/api/services/content-types-info/content-types-info.service.ts

```diff
@@ -31,7 +31,7 @@ export class ContentTypesInfoService {
         {
             clazz: 'com.dotcms.contenttype.model.type.ImmutablePersonaContentType',
             icon: 'fa-user',
-            label: 'persona'
+            label: 'dot-persona'
```

Comment:
Why this change? none of the other have dot- prefix.

Reply:
this is wrong I will rever it it, was because a "reference change" Westorm did.